httpserver-rs/src/lib.rs

```diff
@@ -263,16 +273,38 @@ impl HttpServer {
             &ld.actor_id,
             &req.path
         );
-        let tx = ProviderTransport { bridge, ld: &ld };
+        let tx = ProviderTransport::new_with_timeout(&ld, Some(bridge), Some(timeout));
         let ctx = wasmbus_rpc::Context::default();
         let actor = HttpServerSender::via(tx);
-        let resp = actor.handle_request(&ctx, &req).await?;
-        trace!(
-            "received from actor {}: code={}",
-            &ld.actor_id,
-            &resp.status_code
-        );
-        Ok(resp)
+        match actor.handle_request(&ctx, &req).await {
+            Err(RpcError::Timeout(_)) => {
+                error!(
+                    "actor {} req path {} timed out: returning 503",
+                    &ld.actor_id, &req.path,
+                );
+                Ok(HttpResponse {
+                    status_code: 503,
+                    body: b"Actor took too long or was unavailable.".to_vec(),
```

Comment:
I know this seems like a super nitpick, but for security reasons, maybe we shouldn't disclose that there's an actor on the other side of this provider. Maybe just "No response within timeout" or something like that.

Reply:
Getting this error message would immediately tell a consumer they've discovered a wasmCloud installation.

Reply:
good point